data/helpers.d/network

```diff
@@ -0,0 +1,15 @@
+# Check the path syntax
+# Check the slash at the beginning of path and its absence at ending
+#
+# example: ynh_check_path
+#
+# usage: ynh_check_path
+ynh_check_path () {
+	test -n "$path" || ynh_die "The path variable is empty."
```

Comment:
Should we define "/" like default value ?

Est ce qu'on devrait définir / comme valeur par défault ?